qctrlopencontrols/driven_controls/predefined.py

```diff
@@ -1206,3 +1207,128 @@ def new_modulated_gaussian_control(
         detunings=np.zeros(segment_count),
         durations=np.array([segment_duration] * segment_count),
     )
+
+
+def new_drag_control(
+    rabi_rotation: float,
+    segment_count: int,
+    duration: float,
+    width: float,
+    beta: float,
+    name: Optional[str] = None,
```

Comment:
```suggestion
    azimuthal_angle: float = 0.0,
    name: Optional[str] = None,
```

Reply:
Add azimuthal_angle here to bring in line with other pulses definitions. [@tiborr this is to allow users to define the axis around which the DRAG pulse rotates the qubit state - imagine a Bloch sphere and say you're doing a pi/2 pulse, azimuthal_angle will determine the point where you land on the equator]

Reply:
Ok, thanks. But I think I have to make a few additional changes so the tests don't fail :)

Reply:
@tiborr yes of course, you'll naturally need a follow through on this in the tests :)